etc/desi.json

```diff
@@ -115,6 +118,16 @@
         "svdc-summer2018":{
             "svdc-summer2018/.*$":"svdc-summer2018.tar"
         },
+        "svdc-summer2019":{
+            "svdc-summer2019/svdc2019([a-z])/(docs|fiberassign|fiberassign2|survey)/.*$":"svdc-summer2019/svdc2019\\1/svdc2019\\1_\\2.tar",
+            "svdc-summer2019/svdc2019([a-z])/spectro/redux/v([0-9]+)/[^/]+$":"svdc-summer2019/svdc2019\\1/spectro/redux/v\\2/svdc2019\\1_spectro_redux_v\\2_files.tar",
+            "svdc-summer2019/svdc2019([a-z])/spectro/redux/v([0-9]+)/(QA|calibnight|preproc|redrock|run)/.*$":"svdc-summer2019/svdc2019\\1/spectro/redux/v\\2/svdc2019\\1_spectro_redux_v\\2_\\3.tar",
+            "svdc-summer2019/svdc2019([a-z])/spectro/redux/v([0-9]+)/(exposures|spectra-64|tiles)/[^/]+$":"svdc-summer2019/svdc2019\\1/spectro/redux/v\\2/\\3/svdc2019\\1_spectro_redux_v\\2_\\3_files.tar",
+            "svdc-summer2019/svdc2019([a-z])/spectro/redux/v([0-9]+)/(exposures|spectra-64|tiles)/([0-9]+)/.*$":"svdc-summer2019/svdc2019\\1/spectro/redux/v\\2/\\3/svdc2019\\1_spectro_redux_v\\2_\\3_\\4.tar",
+            "svdc-summer2019/svdc2019([a-z])/spectro/sim/v([0-9]+)/([0-9]{8})/.*$":"svdc-summer2019/svdc2019\\1/spectro/sim/v\\2/svdc2019\\1_spectro_sim_v\\2_\\3.tar",
```

Comment:
`spectro/redux/{specprod}` and `spectro/sim/{pixprod}` happen to be `v[0-9]+` for now, but could be more general.  Let's make those `[0-9a-zA-Z_-]+` like other spectro specprod and pixprod regexes.  Otherwise this looks good.

Reply:
Good catch, fixed!